### ISSUE TITLE
How to get Allegro Pay fee?

### ISSUE THREAD

**Issue by Mujoku** [allegro/allegro-api#9174]
Is there a way to retrieve Allegro Pay fee from the API?

**Comment by MartaNowaczyk** [comment]
When you retrieve the history of billing operations [GET /billing/billing-entries](https://developer.allegro.pl/documentation/#operation/getBillingEntries), for “type.id” we return “APF” ("Fee for providing the Allegro Pay payment method"). 
You will check all types of billing operations via [GET /billing/billing-types](https://developer.allegro.pl/documentation/#operation/getBillingTypes).

**Comment by Mujoku** [comment]
Thank you Marta for reaching out! Let me clarify. What I'm trying to achieve here is to calculate an APF in similar fashion to that of **/pricing/offer-fee-preview**. A **preview of an APF** is what I'm trying to retrieve. Is there a way to do that?

**Comment by MartaNowaczyk** [comment]
No, and the fee is only charged when the customer pays for the order via Allegro Pay, so there will be no such thing in the offer fees.
